server/proxy/grpcproxy/cluster.go

```diff
@@ -79,38 +82,36 @@ func NewClusterProxy(lg *zap.Logger, c *clientv3.Client, advaddr string, prefix
 	return cp, donec
 }
 
-func (cp *clusterProxy) resolve(prefix string) {
+func (cp *clusterProxy) establishEndpointWatch(em endpoints.Manager, prefix string) {
```

Comment:
So, `establishEndpointWatch` is a method and we create `em endpoints.Manager` only once, right? Think `em endpoints.Manager` is better defined as a field inside `cp *clusterProxy`?

Reply:
Done. PTAL.